src/resources/competition/controller.js

```diff
@@ -1,3 +1,4 @@
+const { competition } = require("../../utils/database");
```

Comment:
Should this be here? Looking at your `database.js` file, you're only exporting prisma there